java/src/main/java/com/codurance/training/taskupdater/TaskUpdater.java

```diff
@@ -0,0 +1,24 @@
+package com.codurance.training.taskupdater;
+
+import com.codurance.training.tasks.Task;
+
+import java.io.PrintWriter;
+import java.util.List;
+import java.util.Map;
+
+public class TaskUpdater implements TaskUpdaterInterface{
+    @Override
+    public void updateTask(String idString, boolean done,Map<String, List<Task>> tasks, PrintWriter out) {
```

Comment:
store Map<String, List<Task>> tasks as a static variable somewhere and assess that variable instead of passing it to all the function , this applies for all the places where Map<String, List<Task>> tasks is being passed as argument

Reply:
Updated the above changes